lib/driver.js

```diff
@@ -573,18 +573,24 @@ class XCUITestDriver extends BaseDriver {
     this.lifecycleData.createSim = false;
 
     // if we get generic names, translate them
-    this.opts.deviceName = (function translateDeviceName (dn = '') {
+    this.opts.deviceName = (function translateDeviceName (pv, dn = '') {
       let deviceName = dn;
       if (dn.toLowerCase() === 'iphone simulator') {
         deviceName = 'iPhone 6';
       } else if (dn.toLowerCase() === 'ipad simulator') {
-        deviceName = 'iPad Retina';
+        if (parseFloat(pv, 10) < 10.3) {
```

Comment:
`parseFloat` doesn't take a base parameter, so the `10` is superfluous. do we know at this point what the type of `pv` is? and does it still work if it is '10.1.2' for example?